cmd/influxd/server_integration_test.go

```diff
@@ -419,3 +419,90 @@ func Test_Server5NodeIntegration(t *testing.T) {
 
 	simpleQuery(t, testName, nodes[:1], `select value from "foo"."bar".cpu`, expectedResults)
 }
+
+func Test_AllTagCombinationsInShowSeriesAreSelectable(t *testing.T) {
+	nNodes := 1
+	basePort := 8390
+	testName := "all tag combinations in SHOW SERIES are selectable"
+	now := time.Now().UTC()
+	nodes := createCombinedNodeCluster(t, testName, nNodes, basePort)
+
+	createDatabase(t, testName, nodes, "foo")
+	createRetentionPolicy(t, testName, nodes, "foo", "bar")
+
+	bp := influxdb.BatchPoints{
+		Database: "foo",
+	}
+
+	measurementName := "my_measurement"
+
+	// 21 points, 3 tagRarity results in this test failing ~100% of the time
```

Comment:
Also, when we do fix the issue, these comments will be inaccurate. We should just remove them, and not merge this test until the test passes.